lib/webpacker_uploader/instance.rb

```diff
@@ -3,26 +3,37 @@
 require "mime-types"
 
 class WebpackerUploader::Instance
-  # TODO make this configurable
-  IGNORE_EXTENSION = %w[.map].freeze
-  public_constant :IGNORE_EXTENSION
+  DEFAULT_IGNORE_EXTENSION = %w[.map].freeze
```

Comment:
I would completely remove the `IGNORE_EXTENSION` and `DEFAULT_IGNORE_EXTENSION` constants. We can directly use the `ignored_extensions` class attribute below.